pennylane/templates/subroutines/double_excitation_unitary.py

```diff
@@ -429,8 +421,13 @@ def DoubleExcitationUnitary(weight, wires=None):
 
     Args:
         weight (float): angle :math:`\theta` entering the Z rotation acting on wire ``p``
-        wires (Iterable or Wires): Wires ``s, r, q, p`` that the template acts on. Must be of length 4.
-            Accepts an iterable of numbers or strings, or a Wires object.
+        wires1 (Iterable or Wires): Wires of the qubits representing the first particle-hole pair.
+            The first wire is interpreted as ``s`` and the last wire is interpreted as ``r``. Wires in between
+            represent the particles/holes in between the pair.
+            Must be of minimum length 2.
+        wires2 (Iterable or Wires): Wires of the qubits representing the second particle-hole pair.
+            The first wire is interpreted as ``q`` and the last wire is interpreted as ``p``.
+            Must be of minimum length 2.
```

Comment:
It would be great to have a better way to explain here. I am also not sure, do s,r represent the first ph pair?

Reply:
Fantastic!